File: code/server/src/Core/Entities/EmpresaMantenimiento.cs
using IMT_Reservas.Server.Core.Abstraction;

namespace IMT_Reservas.Server.Core.Entities;

public class EmpresaMantenimiento : Entity
{
    public string Nombre { get; set; } = string.Empty;
    public string? Nit { get; set; }
    public string? Direccion { get; set; }
    public string? Telefono { get; set; }
    public string? NombreResponsable { get; set; }
    public string? ApellidoResponsable { get; set; }
}

File: code/client/src/app/entities/admin/model/empresa-mantenimiento.ts
import { BaseModel } from '@shared/model';
export class EmpresaMantenimiento extends BaseModel {
  NombreEmpresa?: string | null;
  NombreResponsable?: string | null;
  ApellidoResponsable?: string | null;
  Telefono?: string | null;
  Nit?: string | null;
  Direccion?: string | null;
  constructor() {
    super();
    this.NombreEmpresa = null;
    this.NombreResponsable = null;
    this.ApellidoResponsable = null;
    this.Telefono = null;
    this.Nit = null;
    this.Direccion = null;
  }
}
